base/logger_console.go

```diff
@@ -74,6 +75,14 @@ func (lcc *ConsoleLoggerConfig) init() error {
 	// Always enable the HTTP log key
 	lcc.LogKeys = append(lcc.LogKeys, logKeyNames[KeyHTTP])
 
+	// Overriding to debug log level if we see any "+" log keys
+	for _, k := range lcc.LogKeys {
+		if strings.HasSuffix(k, "+") && !lcc.LogLevel.Enabled(LevelDebug) {
+			Infof(KeyAll, "Found \"+\" Log Key. Overriding to debug log level.")
+			lcc.LogLevel.Set(LevelDebug)
+		}
+	}
+
```

Comment:
Does this mean that if a console config has logKeys:["Changes", "CRUD+"], we're going to end up logging Changes+?

That may not be the end of the world, given that the general intention is to move away from the log key approach.  It would be a difference between 2.0 and 2.1 functionality, though, and might need a mention in the release notes.

Reply:
Yep, we're effectively dropping a level of granularity in 2.1.0 here.